### PR TITLE
#226 - Restrict accepted mimetypes in file widget

### PR DESCRIPTION
**Pull request by gr8pathik**
### Reasons for making this change
- Added a feature to support accepted mime type in file widget

If this is related to existing tickets, include links to them as well.
#226 - Restrict accepted mime types in file widget
### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://react-jsonschema-form.readthedocs.io/en/latest/#contributing) of the Markdown text I've added
* [x] **I'm adding or updating code**
  - [ ] I've added and/or updated tests
  - [ ] I've updated [docs](https://react-jsonschema-form.readthedocs.io/) if needed
  - [x] I've run `npm run cs-format` on my branch to conform my code to [prettier](https://github.com/prettier/prettier) coding style
* [x] **I'm adding a new feature**
  - [x] I've updated the playground with an example use of the feature

### CODE REVIEW

**Comment by epicfaace** [comment]
@gr8pathik looks good, can you add a test?

**Comment by gr8pathik** [comment]
> @gr8pathik looks good, can you add a test?

@epicfaace I don't think this can have any test. Because we cannot use the accept attribute as a validation tool. It is just used to hint to browsers to only show files that are allowed for the current input.

**Comment by epicfaace** [comment]
@gr8pathik I just meant that you could make a test like the one here -- https://github.com/mozilla-services/react-jsonschema-form/blob/679b38115eeb0e808ced6d39fcf3da3b9721b69c/test/ObjectField_test.js#L185 -- just to see if the widget with the proper attribute is rendered.


Also, I believe the uiSchema in the playground should be changed from `"ui:accept": ".pdf"` to `"ui:options": {"accept": ".pdf"}`

**Comment by gr8pathik** [comment]
@epicfaace Added the test cases.

**Comment by epicfaace** [comment]
@gr8pathik thanks! Can you add a section for documentation of this option over here: https://react-jsonschema-form.readthedocs.io/en/latest/form-customization/#file-widgets

**Comment by gr8pathik** [comment]
@epicfaace I have updated the documentation.

File: docs/form-customization.md [hunk 1]
```diff
@@ -231,6 +231,28 @@ The included `FileWidget` exposes a reference to the `<input type="file" />` ele
 
 This allows you to programmatically trigger the browser's file selector, which can be used in a custom file widget.
 
+#### File widget options
+
+##### `accept` option
+
+You can use accept attribute to specify a filter for what file types the user can pick from the file input dialog box
```
Comment (epicfaace) [inline docs/form-customization.md]:
```suggestion
You can use accept attribute to specify a filter for what file types the user can upload.
```

**Comment by epicfaace** [comment]
> Thanks! Can you accept my suggestion -- and then we should be good to go.

@gr8pathik ?

File: docs/form-customization.md (hunk 1 above)
Comment (gr8pathik) [reply docs/form-customization.md]: Done

**Comment by gr8pathik** [comment]
@epicfaace is there any issue with this PR or we are good to merge?

File: docs/form-customization.md [hunk 2]
```diff
@@ -231,6 +231,28 @@ The included `FileWidget` exposes a reference to the `<input type="file" />` ele
 
 This allows you to programmatically trigger the browser's file selector, which can be used in a custom file widget.
 
+#### File widget options
+
+##### `accept` option
+
+You can use accept attribute to specify a filter for what file types the user can upload
```
Comment (epicfaace) [inline docs/form-customization.md]:
```suggestion
You can use the accept attribute to specify a filter for what file types the user can upload:
```

**Comment by solomontang** [comment]
When should we expect the next release to land with this added?

**Comment by epicfaace** [comment]
Once #1419 is merged

**Pull request merged**